src/integrationTest/java/com/mongodb/hibernate/query/select/SimpleSelectQueryIntegrationTests.java

```diff
@@ -0,0 +1,186 @@
+                q -> q.setParameter(1, Country.USA.name()),
+                List.of(2, 3, 4)));
+    }
+
+    @Test
+    void testComparisonByLT(SessionFactoryScope scope) {
+        scope.inTransaction(session -> assertContactQueryResult(
+                session, "from Contact where age < :age", q -> q.setParameter("age", 35), List.of(1, 3, 5)));
+    }
+
+    @Test
+    void testComparisonByLTE(SessionFactoryScope scope) {
+        scope.inTransaction(session -> assertContactQueryResult(
+                session, "from Contact where age <= ?1", q -> q.setParameter(1, 35), List.of(1, 2, 3, 5)));
+    }
+
+    @Test
+    void testComparisonByGT(SessionFactoryScope scope) {
+        scope.inTransaction(session -> assertContactQueryResult(
+                session, "from Contact where age > :age", q -> q.setParameter("age", 18), List.of(2, 4, 5)));
+    }
+
+    @Test
+    void testComparisonByGTE(SessionFactoryScope scope) {
+        scope.inTransaction(session -> assertContactQueryResult(
+                session, "from Contact where age >= :age", q -> q.setParameter("age", 18), List.of(1, 2, 4, 5)));
+    }
+
+    @Test
+    void andFilterTest(SessionFactoryScope scope) {
+        scope.inTransaction(session -> assertContactQueryResult(
+                session,
+                "from Contact where country = ?1 and age > ?2",
+                q -> q.setParameter(1, Country.CANADA.name()).setParameter(2, 18),
+                List.of(2, 4)));
+    }
+
+    @Test
+    void orFilterTest(SessionFactoryScope scope) {
+        scope.inTransaction(session -> assertContactQueryResult(
+                session,
+                "from Contact where country = :country or age > :age",
+                q -> q.setParameter("country", Country.CANADA.name()).setParameter("age", 18),
+                List.of(2, 3, 4, 5)));
+    }
+
+    @Test
+    void testFieldNonNumericLiteralValue(SessionFactoryScope scope) {
+        scope.inTransaction(session ->
+                assertContactQueryResult(session, "from Contact where country = 'USA'", null, List.of(1, 5)));
+    }
+
+    @Test
+    void testFieldNumericLiteralValue(SessionFactoryScope scope) {
+        scope.inTransaction(
+                session -> assertContactQueryResult(session, "from Contact where age < 35", null, List.of(1, 3, 5)));
+    }
+
+    @Test
+    void testFieldStringLiteralValue(SessionFactoryScope scope) {
+        scope.inTransaction(session ->
+                assertContactQueryResult(session, "from Contact where country != 'USA'", null, List.of(2, 3, 4)));
+    }
+
+    @Test
+    void testNotFilterOperation(SessionFactoryScope scope) {
+        scope.inTransaction(session -> assertContactQueryResult(
+                session, "from Contact where age > 18 and not (country = 'USA')", null, List.of(2, 4)));
+    }
+
+    @Test
+    void testProjectOperation(SessionFactoryScope scope) {
+        scope.inTransaction(session -> {
+            var results = session.createSelectionQuery(
```

Comment:
Is new translation code for this test included in the PR? I didn't notice it.

Reply:
No new translation code is needed for the previous `loading by primary key` has finished that part. I only added this test for it is a common usage of select query.